KachnaOnline.App/Controllers/EventsController.cs

```diff
@@ -144,6 +150,164 @@ public async Task<ActionResult<ManagerEventDto>> PlanEvent(BaseEventDto newEvent
             }
         }
 
+        /// <summary>
+        /// Returns a list of <see cref="StateDto"/> of conflicting planned states for event with the given ID.
+        /// </summary>
+        /// <param name="eventId">ID of the event to get the conflicting planned states for.</param>
+        /// <returns>A list of <see cref="EventDto"/>.</returns>
+        /// <response code="200">The list of conflicting states.</response>
+        /// <response code="404">No such event exists.</response>
+        [HttpGet("{eventId}/conflicting_states")]
+        [ProducesResponseType(typeof(IEnumerable<StateDto>), StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        public async Task<ActionResult<IEnumerable<StateDto>>> GetConflictingPlannedStatesForEvent(int eventId)
+        {
+            try
+            {
+                return this.Ok(await _eventsFacade.GetConflictingStatesForEvent(eventId));
+            }
+            catch (EventNotFoundException)
+            {
+                return this.NotFoundProblem("The specified event does not exist.");
+            }
+        }
+
+        /// <summary>
+        /// Get states linked to the event specified by <paramref name="eventId"/>.
+        /// </summary>
+        /// <param name="eventId">An ID of the event to link planned states to.</param>
+        /// <returns>An <see cref="EventDto"/> with list of <see cref="StateDto"/> as linked planned states.</returns>
+        /// <response code="200">The planned states were linked to the event.</response>
+        /// <response code="404">The event with the given ID <paramref name="eventId"/> does not exist.</response>
+        /// <response code="409">The event from the past cannot be modified.</response>
+        [AllowAnonymous]
+        [HttpGet("{eventId}/linked_states")]
+        [ProducesResponseType(typeof(EventDto), StatusCodes.Status200OK)]
```

Comment:
This should be `typeof(IEnumerable<EventDto>)`.

Reply:
Yes, the controller documentation and its method decorators are still work-in-progress a bit. I am working on updating these right now.